### DISCUSSION TITLE
Domain Setup and Authentication?

### DISCUSSION THREAD

**Discussion by nicecraftz** [plan-player-analytics/Plan#3308] [Q&A]
Hello everyone, I just set up plans on my network and it is working fine, tho I would like to connect It to the domain to make it better looking and also add authentications for us owners who want to check the data, I generated my SSL certificate using Certbot and I also have Linux and I don't have problems with firewall as I already set up the port, thanks in advance! PS i run Apache with WordPress

**Comment by AuroraLS3** [comment]
You need an A record in your DNS. After that setup Plan Alternative_IP settings as the address of the domain.

**Reply by nicecraftz** [reply]
wouldn't I need to set up SSL first etc?

**Reply by AuroraLS3** [reply]
You can do either one first. I assumed you had already done the SSL since you mentioned a generated cert.
